docker/dev_vpp_agent/Dockerfile

```diff
@@ -77,15 +68,7 @@ WORKDIR /root/
 COPY supervisord.conf /etc/supervisord/supervisord.conf
 
 # and default plugins conf
-COPY defaultplugins.conf /opt/vpp-agent/dev/defaultplugins.conf
-
-# and linux plugin conf
-COPY linuxplugin.conf /opt/vpp-agent/dev/linuxplugin.conf
-
-# and logs conf
-COPY logs.conf /opt/vpp-agent/dev/logs.conf
-
-
+COPY defaultplugins.conf linuxplugin.conf logs.conf /opt/vpp-agent/dev/
```

Comment:
This is not recommended (https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy). However since the building process for this image ignores build cache (hence `--no-cache` in `build.sh`), it doesn't matter.